packages/gatsby-cli/src/util/version.ts

```diff
@@ -1,8 +1,8 @@
-const { setDefaultTags } = require(`gatsby-telemetry`)
-const path = require(`path`)
+import { setDefaultTags } from "gatsby-telemetry"
+import path from "path"
 
-exports.getLocalGatsbyVersion = () => {
-  let version
+export const getLocalGatsbyVersion = (): string => {
+  let version = ``
```

Comment:
could I actually make a request here? We moved this functionality to `gatsby-core-utils`, but the gatsby-telemetry should not live in there. Could we make this function essentially just call the fn from gatsby-core-utils`?